src/plugins/query_enhancements/public/plugin.tsx

```diff
@@ -62,7 +62,7 @@ export class QueryEnhancementsPlugin
         startServices: core.getStartServices(),
         usageCollector: data.search.usageCollector,
       }),
-      getQueryString: (currentQuery: Query) => `source = ${currentQuery.dataset?.title}`,
+      getQueryString: (currentQuery: Query) => `source = ${currentQuery.dataset?.title} | head 10`,
```

Comment:
we had to remove this because PPL head 10 will cause some null pointer exception based on the time filter

Reply:
okay, will remove it for as discussed with the team but this causes queries to run for a long time and fail as they try to fetch lot of content without the limit